app/create_db.py

```diff
@@ -1,5 +1,4 @@
-#!/usr/bin/env python
-
+import regex
```

Comment:
This takes us up to two imports of regex in the same file

Reply:
Yeah that shouldn't be there! I may have messed that up while handling some merge conflicts as there were a lot in our import sections.